configure

```diff
@@ -7993,5 +8012,4 @@ fi
 
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: Good, the configure finished." >&5
 printf "%s\n" "$as_me: Good, the configure finished." >&6;}
-printf "%s\n"
-
+printf "%s\n"
```

Comment:
This part should be reverted, i.e. don't edit the output manually.

Reply:
will address later

Reply:
Are you going to handle it soon?

Reply:
pushed changes, hope all ok now.